server/container/util-java8/src/test/java/org/apache/james/util/CommutativityChecker.java

```diff
@@ -0,0 +1,72 @@
+/****************************************************************
```

Comment:
This commit should be before the commit before it.

Reply:
The github order is messed up due to rebase. Sorry.